components/Forms/RegistrationForm/RegistrationForm.js

```diff
@@ -114,18 +117,20 @@ function RegistrationForm({ initialValues, onSubmit, onSuccess }) {
       validationSchema={registrationSchema}
     >
       {({ isSubmitting }) => (
-        <Form className="w-full flex items-center flex-col [&>p]:my-5 [&>p]:mx-auto">
-          <p>
-            We work closely with military veterans, service members, and spouses who are passionate
-            about transitioning into the tech industry. We work with over 7,000 members who are all
-            working towards relevant goals on Slack and in-person meet-ups. Membership is free!
-            Unfamiliar with Slack?{` `}
-            <Link href="/slack_guide">
-              <a>Learn how to use it!</a>
-            </Link>
-          </p>
-
-          <div className="flex flex-col justify-between w-full max-w-lg">
+        <Form className="flex flex-col items-center w-full">
+          <div className="max-w-xl px-4">
+            <p className="pb-4">
```

Comment:
```suggestion
            <p className="max-w-xl px-4 pb-4">
```

do we need the extra element?